docs/debugging.md

```diff
@@ -120,7 +120,7 @@ Custom debugger commands executed this way should be short-lived processes, and
 You can use Safari to debug the iOS version of your app without having to enable "Debug JS Remotely".
 
 - On a physical device go to: `Settings → Safari → Advanced → Make sure "Web Inspector" is turned on` (This step is not needed on the Simulator)
-- On your Mac enable Develop menu in Safari: `Preferences → Advanced → Select "Show Develop menu in menu bar"`
+- On your Mac enable Develop menu in Safari: `Settings (or Preferences) → Advanced → Select "Show Develop menu in menu bar"`
```

Comment:
```suggestion
- On your Mac enable Develop menu in Safari: `Settings... (or Preferences...) → Advanced → Select "Show Develop menu in menu bar"`
```

Label in Safari also include ellipsis, so let's add the dots here too. 🙂 

Btw. We probably also might want to reformat steps in the whole list, so they use font weight instead of code block, but this can be done in the future, as a separate change.